parachain/runtime/tests/test_nomination.rs

```diff
@@ -266,71 +255,6 @@ fn integration_test_operator_withdrawal_can_force_refund_nominators() {
     });
 }
 
-#[test]
-fn integration_test_liquidating_operators_for_low_collateralization_also_liquidates_nominators() {
```

Comment:
I suppose a similar test would not to be re-added, but to check that the lazy update was successful instead of checking the iteration